cva6/regress/install-cva6.sh

```diff
@@ -67,16 +77,20 @@ echo $CVA6_PATCH
 
 if ! [ -d core-v-cores/cva6 ]; then
   git clone --recursive $CVA6_REPO -b $CVA6_BRANCH core-v-cores/cva6
-  cd core-v-cores/cva6; git checkout $CVA6_HASH;
+  git --git-dir core-v-cores/cva6 checkout $CVA6_HASH;
```

Comment:
```suggestion
  git -C core-v-cores/cva6 checkout $CVA6_HASH;
```

`--git-dir` is to use the current repository, but using another `.git/` directory. Here you want to act on another repository so you want option `-C` instead.

Reply:
My bad, I will correct it :smile: